cohorts/load.py

```diff
@@ -1297,6 +1297,10 @@ def compare_provenance(
     -----------
     Number of discrepancies (0: None)
```

Comment:
This is clarifying that a 0 is returned (as the number of discrepancies) when there are no discrepancies.